app/javascript/components/TripFormModal/TripFormInfo.js

```diff
@@ -139,6 +141,7 @@ const TripFormInfo = () => {
                 isMulti
                 placeholder="Select from the list or create new"
                 className={styles.selectInput}
+                aria-labelledby="travelPartners"
```

Comment:
Instead of adding an `id` for the label and mapping it to the input with `aria-labelledby`, we can use the `inputId` prop (https://react-select.com/props#select-props) which will work with the existing `for` attribute on the label.
```
 inputId={FIELD_NAMES.TRAVEL_PARTNERS}
```

Reply:
This did not seem to work with cypress selector findByLabelText, which I assume should work if it's accessible.